GeoDaCenter/chicago-environment-explorer
# feat: update staff listing 2024

## Problem
Staff listing is out-of-date for 2024

Fixes #149 

## Approach
* fix: fixed my own listing on this view (picture, degrees, title, etc)
* fix: changed Dajoin to MA Student

If any other changes or additions are needed, feel free to comment here :smile:

![Screenshot 2024-05-29 at 5 54 02 PM](https://github.com/GeoDaCenter/chicago-environment-explorer/assets/1413653/5bb29d52-42ea-403f-905f-b48a54801da6)

## How to Test
1. Navigate to https://deploy-preview-160--chicago-env-explorer.netlify.app/about
    * You should see the updated staff listing here
